### ISSUE TITLE
dpkg: error: PATH is not set

### ISSUE THREAD

**Issue by khangprolxag** [cloudcmd/docker#1]
![image](https://cloud.githubusercontent.com/assets/20451172/20456231/f4b84096-ae26-11e6-9300-c72f77ae4521.png)
hi. please help me this, i try to install java jre by using cloud cmder docker image but it always returns PATH is not set? thank you.

**Comment by coderaiser** [comment]
Looks like you should install `apt-utils`. What command do you use to install `java re`?

**Comment by khangprolxag** [comment]
thanks for reply me, i tried to install apt-utils but it still not working 
my commands:
apt-get update
apt-get install apt-utils
![image](https://cloud.githubusercontent.com/assets/20451172/20471803/75cde68e-af69-11e6-8f04-eb6a4e4d1515.png)

**Comment by coderaiser** [comment]
Looks like it is something with `PATH` variable and `dpkg`. But with `PATH` everything is OK:

```sh
/usr/src/app> echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Try from OS console connect to container with:

```
docker exec -it container bash
```

And install what you need.

Also this should work:

```sh
PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" apt-get install apt-utils -y
```

Looks like it is an issue with [console-io](https://github.com/cloudcmd/console).

**Comment by coderaiser** [comment]
Fixed with https://github.com/cloudcmd/console/commit/6902ed80e350260b7e6b9e0e2ffd63137c75f973

**Comment by coderaiser** [comment]
Landed in [v5.10.1](https://github.com/coderaiser/cloudcmd/releases/tag/v5.10.1).
